src/types/formTypes.ts

```diff
@@ -109,15 +113,15 @@ export const institutionDetailsApiTypeSchema = z.object({
       z.number({
         invalid_type_error: IdZodSchemaErrors.parent_rssd_idNumber,
       }),
-      z
-        .string()
-        .regex(/^\d+$|^$/, { message: IdZodSchemaErrors.parent_rssd_idRegex }),
```

Comment:
Looks like this regex also allows empty string, so replacing it with the regtech-regex one causes a bug where I can't submit an Update your financial institution form with a blank RSSD ID ("Hey CFPB, remove this RSSD ID"). 

<img width="843" alt="Screenshot 2024-12-20 at 1 58 24 PM" src="https://github.com/user-attachments/assets/18a1ca04-8999-40af-bd30-11f88a99636c" />

Compare this with trying to submit with a blank TIN. Maybe we can use the regtech-regex regex but make this field nullable/optional?

Reply:
Should be taken care of.